metaflow/main_cli.py

```diff
@@ -123,6 +124,21 @@ def status():
     for flow in Metaflow():
         echo('* %s' % flow, fg='cyan')
 
+@main.command(help='Reset local data store. This will remove all information and artifacts from your runs.')
+def reset_local():
+    # Get the local data store path
+    path = LocalDataStore.get_datastore_root_from_config(echo, create_on_absent=False)
+    
+    if path:
+        if click.confirm("This will reset your local data store." +\
+                        click.style("\nWARNING:", fg='red') +\
+                                    "This step cannot be undone!\nProceed to delete: " +\
+                                    click.style("\"{0}\"".format(path), fg='red') + "?"):
+            shutil.rmtree(path)
+    else:
+        # Nothing to do here since we didnt find an existing store.
+        pass
```

Comment:
Please display a relevant message to the user even if there was nothing to do. I think it is very surprising once you execute a command and see nothing happen.

See `configure reset` if you want ideas on how we handle it for configuration.